nixos/modules/virtualisation/containers.nix

```diff
@@ -164,7 +172,7 @@ in
   };
 
 
-  config = mkIf (!config.boot.isContainer) {
+  config = mkIf (config.boot.enableContainer) {
```

Comment:
Thanks, fixing it!
On Jan 24, 2015 11:29 PM, "Wout Mertens" <notifications@github.com> wrote:

> In nixos/modules/virtualisation/containers.nix
> <https://github.com/NixOS/nixpkgs/pull/5939#discussion_r23499978>:
>
> > @@ -164,7 +172,7 @@ in
> >    };
> >
> >
> > -  config = mkIf (!config.boot.isContainer) {
> > +  config = mkIf (config.boot.enableContainer) {
>
> err... enableContainer vs enableContainer*s*
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NixOS/nixpkgs/pull/5939/files#r23499978>.
>